### PR TITLE
CNV-36170: MultiNetworkPolicy list and details page

### PR DESCRIPTION
**Pull request by upalatucci**
Screenshot from 2024-03-19 15-42-22
Screenshot from 2024-03-19 15-42-17

image

### CODE REVIEW

File: src/utils/models/index.ts [hunk 1]
```diff
@@ -24,3 +24,19 @@ export const EndPointSliceModel: K8sModel = {
...
 };
+
+export const MultiNetworkPolicyModel: K8sModel = {
+  abbr: 'MNP',
+  apiGroup: 'k8s.cni.cncf.io',
+  apiVersion: 'v1beta1',
+  id: 'multinetworkpolicy',
+  kind: 'MultiNetworkPolicy',
+  label: 'multi-networkpolicy',
+  // t('multi-networkpolicy')
```
Comment (metalice) [inline src/utils/models/index.ts:35]: we really need it?

Comment (upalatucci) [reply src/utils/models/index.ts:35]: you mean the translation?

File: src/views/networkpolicies/list/NetworkPolicyPage.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import MultiNetworkPolicyList from './MultiNetworkPolicyList';
+import NetworkPolicyList from './NetworkPolicyList';
+
+export type NetworkPolicyPageNavProps = {
+  namespace: string;
+};
+
+export const trimLastHistoryPath = (pathName: Location['pathname']): string => {
+  return pathName.endsWith('checkups') ? pathName : pathName.replace(/\/[^\/]*$/, '');
```
Comment (metalice) [inline src/views/networkpolicies/list/NetworkPolicyPage.tsx:18]: checkups ?

Comment (upalatucci) [reply src/views/networkpolicies/list/NetworkPolicyPage.tsx:18]: i don't really use this function. Deleted.

Comment (metalice) [reply src/views/networkpolicies/list/NetworkPolicyPage.tsx]: this function for refresh, to stay in same tab, check kif its working without it

File: src/utils/models/index.ts (hunk 1 above)
Comment (metalice) [reply src/utils/models/index.ts:35]: yes

File: src/views/networkpolicies/list/NetworkPolicyPage.tsx (hunk 2 above)
Comment (upalatucci) [reply src/views/networkpolicies/list/NetworkPolicyPage.tsx]: yes works on refresh

**Pull request merged**